cmd/describe/cluster/cmd.go

```diff
@@ -102,6 +102,29 @@ func run(cmd *cobra.Command, argv []string) {
 	var upgradeState *cmv1.UpgradePolicyState
 	var controlPlaneScheduledUpgrade *cmv1.ControlPlaneUpgradePolicy
 
+	// SDN -> OVN Migration Status
+	migrationsStr := ""
+	collection, err := r.OCMClient.FetchClusterMigrations(cluster.ID())
+	if err != nil {
+		r.Reporter.Warnf("Failed to get cluster migrations: %v", err)
+	}
+
+	migrationsToShow := map[string]string{}
+	migrationTypes := map[string]string{}
+	if len(collection.Items().Items()) > 0 {
```

Comment:
If an error did happen does an empty collection gets returned or a nil pointer?

Reply:
No clue, will only run this if there's no error